core/opl/shovel.py

```diff
@@ -1,19 +1,30 @@
 import argparse
 import logging
+import requests
+import json
+import subprocess
+import os
 
-from . import skelet
+from . import skelet, status_data
 
 
 class pluginProw:
     def __init__(self, args):
-        print("Hello from pluginProw init")
+        print("Hello from pluginProw init 2")
```

Comment:
Use `logging....` instead of prints pls.

Reply:
Also these prints created by me can be removed, they were there just for demo purposes